src/composables/ui.ts

```diff
@@ -2,7 +2,7 @@ import { onBeforeUnmount, onMounted, ref } from '@vue/composition-api';
 import { ROUTE_ACCOUNT } from '../popup/router/routeNames';
 
 const homeRouteName = ref(ROUTE_ACCOUNT);
-const isAppActive = ref(false);
+const isAppActive = ref(true);
```

Comment:
We can do it, as it will be set after entering the app - so the initial state should be active

Reply:
This change is bad in my opinion. When user opens up the browser and the recently opened tabs opens this will be set to true, but actually should be false as the tab may not be visible to the user. Instead of setting this to true I suggest to run the `handleVisibilityChange` at the beginning of the `initVisibilityListeners`:

```
function initVisibilityListeners() {
  handleVisibilityChange();
  ...
```